src/main/java/com/sourcegraph/javagraph/AndroidSDKProject.java

```diff
@@ -1,24 +1,53 @@
 package com.sourcegraph.javagraph;
 
-import java.nio.file.FileSystems;
-import java.nio.file.Path;
-import java.util.List;
-import java.util.Set;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.SystemUtils;
+import org.apache.maven.model.building.ModelBuildingException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.*;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.*;
+import java.util.stream.Collectors;
 
 public class AndroidSDKProject implements Project {
```

Comment:
Please add a brief docstring for every class that implements Project and also any other important public classes.